### PR TITLE
add method to call start_accession

### PR DESCRIPTION
**Pull request by peetucket**
## Why was this change made?

to support a new start_accession method, blocked by sul-dlss/dor-services-app#703

when done, return to sul-dlss/technical-metadata-service#71 and follow the checklist there

## Was the documentation (README, API, wiki, consul, etc.) updated?

yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,9 @@ object_client.publish(workflow: 'releaseWF')
 # Shelve an object (push to Stacks)
 object_client.shelve
 
+# Start accessiong an object (intialize assemblyWF or specified workflow, and version object if needed)
```
Comment (mjgiarlo) [inline README.md]: s/accessiong/accessioning/

Comment (peetucket) [reply README.md]: fixed

**Comment by peetucket** [comment]
Codeclimate and rubocop a bit grumpy about the growing size of the object class...

**Comment by jcoyne** [comment]
@peetucket consider putting the accession method into it's own class then?

**Comment by peetucket** [comment]
> @peetucket consider putting the accession method into it's own class then?

Could do that, but in the current design, this method belongs with the object class ... we could split out some methods in a module, which is then included in the object class?

**Comment by jcoyne** [comment]
@peetucket I'm not sure why you think that's the design. There are plenty of other methods that are in their own class for the same reason.  See https://github.com/sul-dlss/dor-services-client/blob/master/lib/dor/services/client/embargo.rb for one example.

**Comment by peetucket** [comment]
> @peetucket I'm not sure why you think that's the design. There are plenty of other methods that are in their own class for the same reason. See [/lib/dor/services/client/embargo.rb@`master`](https://github.com/sul-dlss/dor-services-client/blob/master/lib/dor/services/client/embargo.rb) [](/sul-dlss/dor-services-client/blob/HEAD@{2020-03-02T04:35:49Z}/lib/dor/services/client/embargo.rb) for one example.

indeed, I was getting mixed up with dor-services-app - feel free to withdraw approval and mark as changes requested to block merging until i have a chance to adjust

**Comment by peetucket** [comment]
Ok - separated the accession method into a separate class to reduce the size of the object class, though that doesn't reduce the number of methods, which is codeclimate's complaint.  suggestions?

**Pull request merged**